plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/WooCommercePayments.php

```diff
@@ -73,6 +76,17 @@ public function get_action_label() {
 	 * @return string
 	 */
 	public function get_additional_info() {
+		$data               = WCPayPromotionDataSourcePoller::get_instance()->get_specs_from_data_sources();
```

Comment:
The logic is somewhat duplicate of [getPromotions()](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Internal/Admin/WCPayPromotion/Init.php#L124)

getPromotions() returns an array, so we probably need to add a method in the class to find `woocommerce_payments:woopay`

Reply:
Good catch, fixed on https://github.com/woocommerce/woocommerce/pull/39596/commits/c56335b70b8e5c4ab9685ac0af35ddfd7ea6c5e7.